### PR TITLE
Admin panel frameworks blog

### PR DESCRIPTION
**Pull request by mlhekinci**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

**Test plan (required)**

Demonstrate the code is solid. If not, please add `WIP:` in its title.

**Closing issues**

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

Test documentation! 'MASTER'
[Link to ADMIN-PANEL-FRAMEWORKS-BLOG](https://admin-p-doc-refine.pankod.com)

### CODE REVIEW

File: documentation/blog/2021-11-19-admin-panel-frameworks.md [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+            .then(graphQlDataProvider => setDataProvider(() => graphQlDataProvider));
+    }, []);
+
+    if (!dataProvider) {
+        return <div>Loading < /div>;
+    }
+
+    return (
+        <Admin dataProvider= { dataProvider } >
+            <Resource name="Post" list = { PostList } edit = { PostEdit } create = { PostCreate } />
```
Comment (salihozdemir) [inline documentation/blog/2021-11-19-admin-panel-frameworks.md:142]:
```suggestion
        <Admin dataProvider={dataProvider} >
            <Resource name="Post" list={PostList} edit={PostEdit} create={PostCreate} />
```

File: documentation/blog/2021-11-19-admin-panel-frameworks.md [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+const App = () => {
+
+    const [dataProvider, setDataProvider] = React.useState(null);
+    React.useEffect(() => {
+        buildGraphQLProvider({ clientOptions: { uri: 'http://localhost:4000' } })
+            .then(graphQlDataProvider => setDataProvider(() => graphQlDataProvider));
+    }, []);
+
+    if (!dataProvider) {
+        return <div>Loading < /div>;
```
Comment (salihozdemir) [inline documentation/blog/2021-11-19-admin-panel-frameworks.md:137]:
```suggestion
        return <div>Loading</div>;
```

File: documentation/blog/2021-11-19-admin-panel-frameworks.md [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+Now, when a user browses to /foo or /bar, the components you defined will appear in the main part of the screen.
+
+
+### Data Provider Logic
+When React-admin needs to communicate with API, it uses Data Provider.
+
+<img src={admin_flow} alt="admin" />
+
+Here are the React-Admin data provider methods:
+```
```
Comment (salihozdemir) [inline documentation/blog/2021-11-19-admin-panel-frameworks.md:102]:
```suggestion
```ts
```

Böyle yapsak bu kod alanı renklenir mi acaba?

File: documentation/blog/2021-11-19-admin-panel-frameworks.md [hunk 4]
```diff
@@ -0,0 +1,571 @@
...
+### Data Provider Logic
+
+A data provider is the place where a refine app communicates with an API.
+
+<img src={refine_flow} alt="refine" />
+
+
+A data provider must include following methods:
+
+```
```
Comment (salihozdemir) [inline documentation/blog/2021-11-19-admin-panel-frameworks.md:368]:
```suggestion
```ts
```
Burasıda aynı şekilde renklenir belki

**Review by salihozdemir — COMMENTED** [review]
Great work 🎉

File: documentation/blog/2021-11-19-admin-panel-frameworks.md [hunk 5]
```diff
@@ -0,0 +1,572 @@
...
+* UI: You have full control over the UI elements. Fully customizable, open to use. Works seamlessly with integrated Ant Design System.
+
+* Out-of-the-box: Routing, networking, authentication, state managment, i18n and UI.
+
+* Next.js / SSR integration: refine can be used with Next.js to SSR your pages.
+
+* React Location: React Location router provider support 
+
+### SSR - Next.js Support 
+Refine has support for SSR - Next.js. This is an important feature that distinguishes Refine from other frameworks. Thanks to this feature, Refine provides the opportunity to develop B2C applications in addition to B2B and admin panel.
```
Comment (omeraplak) [inline documentation/blog/2021-11-19-admin-panel-frameworks.md:240]:
```suggestion
Refine has support for SSR - Next.js. This is an important feature that separates Refine from other frameworks. Thanks to this feature, Refine provides the opportunity to develop B2C applications in addition to B2B and admin panel.
```

**Review by omeraplak — CHANGES_REQUESTED** [review]

**Pull request closed without merging**